<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>401</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="start">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>10</height>
       </size>
      </property>
      <property name="text">
       <string>Начало связи:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="start_value">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>10</height>
       </size>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="end">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>10</height>
       </size>
      </property>
      <property name="text">
       <string>Конец связи:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="end_value">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>10</height>
       </size>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
